src/components/ShareImage/docs.md

```diff
@@ -0,0 +1,394 @@
+- `text`
+- `onTextChange`
+- `format`: the metadata of the article's format. The name of a format supported are:
+
+
+## Editorial / Default / No Format
+
+```react
+state: { 
+  fontSize: 60,
+  inverted: false,
+  text: 'Alle meine Ente'
+}
+---
+<div>
+  <ShareImageGenerator
+    fontSize={state.fontSize}
+    onFontSizeChange={(e, value) => setState({ fontSize: Number(value) || '' })}
+    onFontSizeInc={(e, value) => setState({ fontSize: (state.fontSize || 60) + 1})}
+    onFontSizeDec={(e, value) => setState({ fontSize: Math.max(state.fontSize - 1, 0)})}
+    inverted={state.inverted}
+    onInvertedChange={(e, value) => setState({ inverted: value })}
+    text={state.text}
+    onTextChange={e => setState({ text: e.target.value})}
+  />
+  <ShareImagePreview
+    fontSize={state.fontSize}
+    inverted={state.inverted}
+    text={state.text}
+    preview
+  />
+</div>
+```
+
+## Editorial
+```react
+state: { 
+  fontSize: 60,
+  inverted: false,
+  text: '',
+  format: { 
+    color: '#000'
+  }
+}
+---
+<div>
+  <ShareImageGenerator
+    fontSize={state.fontSize}
+    onFontSizeChange={(e, value) => setState({ fontSize: Number(value) || '' })}
+    onFontSizeInc={(e, value) => setState({ fontSize: (state.fontSize || 60) + 1})}
+    onFontSizeDec={(e, value) => setState({ fontSize: Math.max(state.fontSize - 1, 0)})}
+    inverted={state.inverted}
+    onInvertedChange={(e, value) => setState({ inverted: value })}
+    text={state.text}
+    onTextChange={e => setState({ text: e.target.value})}
+    format={state.format}
+  />
+  <ShareImagePreview
+    fontSize={state.fontSize}
+    inverted={state.inverted}
+    text={state.text}
+    format={state.format}
+    preview
+  />
+</div>
+```
+
+## Meta
+```react
+state: { 
+  fontSize: 60,
+  inverted: false,
+  text: '',
+  format: { 
+    title: 'Meta',
+    section: {
+      meta: {
+        title: 'Meta' 
+      }
+    },
+    color: '#000',
+    kind: 'meta'
+  }
+}
+---
+<div>
+  <ShareImageGenerator
+    fontSize={state.fontSize}
+    onFontSizeChange={(e, value) => setState({ fontSize: Number(value) || '' })}
+    onFontSizeInc={(e, value) => setState({ fontSize: (state.fontSize || 60) + 1})}
+    onFontSizeDec={(e, value) => setState({ fontSize: Math.max(state.fontSize - 1, 0)})}
+    inverted={state.inverted}
+    onInvertedChange={(e, value) => setState({ inverted: value })}
+    text={state.text}
+    onTextChange={e => setState({ text: e.target.value})}
+    format={state.format}
+  />
+  <ShareImagePreview
+    fontSize={state.fontSize}
+    inverted={state.inverted}
+    text={state.text}
+    format={state.format}
+    preview
+  />
+</div>
+```
+
+## Formate
+'Format' uses the format.kind value to set the font. If no value is provided it defaults to Rubis.
+
+```react
+state: { 
+  fontSize: 60,
+  inverted: false,
+  text: '',
+  format: { 
+    title: 'Aus der Arena',
+    section: {
+      meta: {
+        title: 'Format' 
+      }
+    },
+    kind: 'scribble'
```

Comment:
```suggestion
    kind: 'scribble',
    color: '#D74132'
```

Reply:
needs a color so that background is correct afaik

Reply:
does it? we use 
`const formatColor = format?.color || colors[format?.kind]`
to determine the color.

Reply:
ok, so that doesn't work in this case. It's a black background without a specific color

Reply:
aaaah for the background color. Got it

Reply:
what if we do 
`  const formatBackgroundColor = format?.color || colors[format?.kind]
`
and
`        backgroundColor: inverted ? formatBackgroundColor : '#FFF',
`

Reply:
ah forget that

Reply:
formatColor is the way to go, thx!

Reply:
pushed a fix